tests/integration_test/services_test.py

```
@@ -6904,6 +6904,312 @@
     CognitionSchema.objects(bot=pytest.bot).delete()
 
 
+@pytest.mark.asyncio
+@responses.activate
+@mock.patch.object(LLMProcessor, "__collection_exists__", autospec=True)
+@mock.patch.object(LLMProcessor, "__create_collection__", autospec=True)
+@mock.patch.object(LLMProcessor, "__collection_upsert__", autospec=True)
+@mock.patch.object(MailUtility,"format_and_send_mail", autospec=True)
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)